test/formatter-oficial.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+var test = require("ava");
+var webpack = require("webpack");
+
+var conf = require("./utils/conf");
+
+test.cb("eslint-loader can use oficial formatter", function(t) {
+  t.plan(1);
+  webpack(
+    conf(
+      {
+        entry: "./test/fixtures/error.js"
+      },
+      {
+        formatter: "table"
+      }
+    ),
+    function(err, stats) {
+      if (err) {
+        throw err;
+      }
+
+      // console.log("### Here is a example of oficial formatter")
+      // console.log(
+      //   "# " +
+      //   stats.compilation.errors[0].message
+      //     .split("\n")
+      //     .join("\n# ")
+      // )
```

Comment:
Need better test

Reply:
The tests with formatters are like this, in fact all tests needs to be improved, I'll do it in a next pr